arduino-ide-extension/src/browser/contributions/verify-sketch.ts

```diff
@@ -65,7 +71,17 @@ export class VerifySketch extends SketchContribution {
     }
 
     async verifySketch(exportBinaries?: boolean): Promise<void> {
+
+        // even with buttons disabled, better to double check if a verify is already in progress
```

Comment:
> better to double check

Why? This code should not run if `isEnabled` is implemented correctly for the command handler. Did you experience a bug?

Reply:
I added this check in order to future-proof the verify-sketch implementation:

https://arduino.atlassian.net/browse/ATL-1045 reports an error (Error: Request upload failed with message: 2 UNKNOWN: exit status 1) when a verify/build is run while another is in progress.

At the moment we only have buttons/menu items, but the check whether or not continue with the action should be inside the action handler itself in my opinion.. 

Do you have concerns about this approach?